lib/authc/AssertionAuthenticationResult.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var nJwt = require('njwt');
+var Account = require('../resource/Account');
+
+function AssertionAuthenticationResult(dataStore, data) {
```

Comment:
I see, but why can't why just use the `AuthenticationResult` instead of creating a new one?